echopop/nwfsc_feat/ingest_nasc.py

```diff
@@ -525,22 +525,53 @@ def merge_echoview_nasc(
     # Return two DataFrames: the complete intervals and the cells that will be integrated
     return df_intervals, merged_exports_df
     
+def read_transect_region_haul_key(
+    filename: Path,
+    sheetname: str,
+    rename_dict: Optional[Dict[str, str]] = None
+) -> pd.DataFrame:
+    """
+    Load the key that maps hauls to export regions to transect numbers.
+    
+    This function reads a CSV or Excel file containing the mapping between
+    transect numbers, region IDs, and haul numbers. It can handle both file
+    formats and allows column renaming.
+    
+    Parameters
+    ----------
+    filename : Path
+        Path to the CSV or Excel file containing the mapping data.
+    sheetname : str
+        Name of the sheet to read (only used for Excel files).
+    rename_dict : Optional[Dict[str, str]], default None
+        Dictionary for renaming columns, where keys are original column names
+        and values are new column names.
+        
+    Returns
+    -------
+    pd.DataFrame
+        DataFrame containing only the columns "transect_num", "region_id", and "haul_num".
+        
+    Notes
+    -----
+    The input file must contain columns that can be mapped to "transect_num",
+    "region_id", and "haul_num", either directly or via the rename_dict.
+    """
 
-# def read_transect_region_file() -> pd.DataFrame:
-#     """
-#     Read external transect region definition file (if provided).
+    # Determine appropriate file reader
+    if filename.suffix == ".csv":
+        transect_region_df = read_csv_file(filename)
+    else:
+        transect_region_df = read_xlsx_file(filename, sheetname)
```

Comment:
I'd suggest just removing these and using `pd.read_csv` and `pd.read_excel` directly since the other line in both functions `export_file.columns = export_file.columns.str.lower()` is identical for both functions and can be easily added here.